packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -43,66 +29,38 @@ const int minimumAndroidSdkVersion = 25;
 /// This should be used over accessing androidSdk.adbPath directly because it
 /// will work for those users who have Android Platform Tools installed but
 /// not the full SDK.
-String getAdbPath([ AndroidSdk existingSdk ]) {
+String getAdbPath(AndroidSdk existingSdk) {
```

Comment:
Removed locating the android sdk, because if it wasn't found before it won't be found again